server/handler/status.go

```diff
@@ -60,10 +61,24 @@ func (h *Status) Handle(ctx context.Context, eventType, deliveryID string, paylo
 	commitSHA := event.GetCommit().GetSHA()
 
 	if sender.GetLogin() != h.PullOpts.AppName+"[bot]" {
-		auditMessage := fmt.Sprintf("Entity '%s' overwrote status check '%s' on ref=%s to status='%s' description='%s' targetURL='%s'", sender.GetLogin(), h.PullOpts.StatusCheckContext, commitSHA, event.GetState(), event.GetDescription(), event.GetTargetURL())
+		auditMessage := fmt.Sprintf(
+			"Entity '%s' overwrote status check '%s' on ref=%s to status='%s' description='%s' targetURL='%s'",
+			sender.GetLogin(),
+			event.GetContext(),
+			commitSHA,
+			event.GetState(),
+			event.GetDescription(),
+			event.GetTargetURL(),
+		)
 		logger.Warn().Str(LogKeyAudit, eventType).Msg(auditMessage)
 
-		err := h.PostStatus(ctx, client, ownerName, repoName, commitSHA, "failure", auditMessage, nil)
+		status := &github.RepoStatus{
```

Comment:
wouldnt this only post 1 context

i guess the alternative is to create a PR object and fill in the data we have, but that feels brittle :( maybe just call the `postGitHubRepoStatus` call twice to at least have consistency?